### ISSUE TITLE
Invalid hash createTokenWithConnectAccount

### ISSUE THREAD

**Issue by wbeuil** [stripe/stripe-ios#1358]
## Summary
Hello guys, I need help to create connect account token with the last iOS-SDK.

With the code below I get an "Invalid hash" error because of the dateOfBirth variable I guess.

Thanks in advance.

## Code to reproduce
```
STPConnectAccountIndividualParams *individual = [STPConnectAccountIndividualParams new];
  individual.firstName = accountData[@"firstName"];
  individual.lastName = accountData[@"lastName"];
  STPConnectAccountAddress *address = [STPConnectAccountAddress new];
  address.line1 = accountData[@"address"][@"line1"];
  address.line2 = accountData[@"address"][@"line2"];
  address.city = accountData[@"address"][@"city"];
  address.state = accountData[@"address"][@"state"];
  address.postalCode = accountData[@"address"][@"postalCode"];
  individual.address = address;
  NSDateComponents *dateOfBirth = [[NSDateComponents alloc] init];
  dateOfBirth.day = 13;
  dateOfBirth.month = 12;
  dateOfBirth.year = 1991;
  individual.dateOfBirth = dateOfBirth;
  individual.idNumber = accountData[@"idNumber"];
  STPConnectAccountParams *accountParams = [[STPConnectAccountParams alloc] initWithIndividual:individual];

[[STPAPIClient sharedClient] createTokenWithConnectAccount:accountParams completion:^(STPToken *token, NSError *error) {
    if (error != nil) {
      reject([NSString stringWithFormat:@"Error"], error.localizedDescription, error);
    } else {
      NSString *accountToken = [NSString stringWithFormat:@"%@", token];
      resolve(accountToken);
    }
  }];
```
## iOS version
iOS 11

## Installation method
Podfile

## SDK version
17.0.0

**Comment by wbeuil** [comment]
Any news guys?

**Comment by yuki-stripe** [comment]
Hi @wbeuil,

Thanks for the report.  I've confirmed this is a bug in the SDK, and we should have a fix in soon.

**Comment by wbeuil** [comment]
@yuki-stripe Thanks for your quick reply.

Should I expect a fix today?

**Comment by yuki-stripe** [comment]
@wbeuil The fix is in master; we plan to release a bugfix version including this within the next few days.

**Comment by davidme-stripe** [comment]
This has been fixed in Stripe SDK 17.0.1.
